pkg/util/kubernetes/kubelet/kubelet.go

```diff
@@ -13,66 +13,70 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
-	"strings"
+	"net/url"
 	"time"
 
+	"github.com/DataDog/datadog-agent/pkg/util/kubernetes"
```

Comment:
imports should be grouped in three tiers: stdlib, 3rd party, local.
`goimports -local github.com/DataDog/datadog-agent` will soon be added to the linting

Reply:
We should add it because it's not the default behavior of goimports.

Reply:
there's a +1 card for that ;)